virtualhosts/etc/e-smith/templates/etc/opt/rh/rh-php71/php-fpm.d/000-virtualhost.conf/10Base

```diff
@@ -56,31 +56,31 @@ php_value[soap.wsdl_cache_dir]  = /var/opt/rh/rh-$PhpRhVersion/lib/php/wsdlcache
         if ($PhpCustomSettings eq 'enabled'){
 
             my $AllowUrlfOpen = $vdb->get_prop("$VhostName",'AllowUrlfOpen') || 'enabled';
-            my $MemoryLimit = $vdb->get_prop("$VhostName",'MemoryLimit') || '0';
-            my $UpMaxFileSize = $vdb->get_prop("$VhostName",'UploadMaxFilesize') || '0';
-            my $PostMaxSize = $vdb->get_prop("$VhostName",'PostMaxSize') || '0';
+            my $MemoryLimit = $vdb->get_prop("$VhostName",'MemoryLimit') || '128';
+            my $UpMaxFileSize = $vdb->get_prop("$VhostName",'UploadMaxFilesize') || '4';
+            my $PostMaxSize = $vdb->get_prop("$VhostName",'PostMaxSize') || '8';
             my $MaxExecTime = $vdb->get_prop("$VhostName",'MaxExecutionTime') || '0';
             my $MaxFileUploads = $vdb->get_prop("$VhostName",'MaxFileUploads') || 'disabled';
             
             $OUT .= "php_admin_flag[allow_url_fopen] = off\n" if ($AllowUrlfOpen eq 'disabled');
-            $OUT .= "php_admin_value[memory_limit] = $MemoryLimit"."M\n" if ($MemoryLimit ne '0');
-            $OUT .= "php_admin_value[upload_max_filesize] = $UpMaxFileSize"."M\n" if ($UpMaxFileSize ne '0');
-            $OUT .= "php_admin_value[post_max_size] = $PostMaxSize"."M\n" if ($PostMaxSize ne '0');
-            $OUT .= "php_admin_value[max_execution_time] = $MaxExecTime\n" if ($MaxExecTime ne 'disabled');
-            $OUT .= "php_admin_value[max_input_time] =  $MaxExecTime\n" if ($MaxExecTime ne 'disabled');
+            $OUT .= "php_admin_value[memory_limit] = $MemoryLimit"."M\n";
+            $OUT .= "php_admin_value[upload_max_filesize] = $UpMaxFileSize"."M\n";
+            $OUT .= "php_admin_value[post_max_size] = $PostMaxSize"."M\n";
+            $OUT .= "php_admin_value[max_execution_time] = $MaxExecTime\n";
+            $OUT .= "php_admin_value[max_input_time] =  $MaxExecTime\n";
             $OUT .= "php_admin_value[max_file_uploads] = $MaxFileUploads\n" if ($MaxFileUploads ne 'disabled');
```

Comment:
Can we handle `max_file_uploads` in a way consistent with other settings (i.e. fall back to ConfigDB `php` props)?

It seems the PHP default for max_file_uploads is `20`:

https://www.php.net/manual/en/ini.core.php#ini.sect.file-uploads

Reply:
```
[root@ns7loc11 ~]# db configuration show php 
php=configuration
    DateTimezone=Europe/Paris
    ExposePhp=
    MaxExecutionTime=2006
    MemoryLimit=280
    PostMaxSize=152
    ShortOpenTag=
    UploadMaxFilesize=76
```
hum we do not have it inside our prop ?

Reply:
Let's add on more ;)